### DISCUSSION TITLE
Anyone with some good examples? I'm having difficulty

### DISCUSSION THREAD

**Discussion by FormerTwitterEmployee** [Extraltodeus/multi-subject-render#12] [General]
There are so many options on this extension that I'm having trouble with getting them to work together well. Would be nice to kinda have a good base to work off from. Regardless, love seeing everyone else's works. 
![01182-horses_in_misty_meadow, 2529478667, 20221125131025](https://user-images.githubusercontent.com/116985596/204060485-3322fa75-e36e-4a73-b58b-57e3a4d46e51.png)
![01341-men_in_suits,_fancy_hotel_ballroom, 2446576236, 20221126071131](https://user-images.githubusercontent.com/116985596/204060498-6a1c2632-f9a8-4593-b6fa-1cdd26264653.png)
![01308-men_in_suits,_fancy_hotel_ballroom, 3046974858, 20221126063956](https://user-images.githubusercontent.com/116985596/204060605-60a65d0f-555d-46d7-9474-03c32b6b221c.png)

**Comment by Extraltodeus** [comment]
Hello!

I am not sure to understand what your first image represents (the vertical one).
your second image needs more steps and denoising strength.

Your third image seems to need more height on the final blend and maybe a prompt that would make the characters less close to the image so they won't be cropped.
